sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/client.py

```diff
@@ -56,75 +56,75 @@
 
 class AMQPClient(object):
     """An AMQP client.
-
-    :param remote_address: The AMQP endpoint to connect to. This could be a send target
-     or a receive source.
-    :type remote_address: str, bytes or ~uamqp.address.Address
-    :param auth: Authentication for the connection. This should be one of the subclasses of
-     uamqp.authentication.AMQPAuth. Currently this includes:
-        - uamqp.authentication.SASLAnonymous
-        - uamqp.authentication.SASLPlain
-        - uamqp.authentication.SASTokenAuth
+# TODO: would it be a target or source?
```

Comment:
Hostname type -> str, pyamqp.endpoint.Source, pyamqp.endpoint.Target? -- I am not sure this is correct